www/helpers.php

```diff
@@ -0,0 +1,7 @@
+<?php
+function idx($array, $key, $default = null){
```

Comment:
I _think_ this function is equivalent to:

```php
return $array[$key] ?? $default;
```

Reply:
yeah, you're right, this helper is a little much and not really necessary